Source/WebGPU/WebGPU/Device.mm

```diff
@@ -497,6 +502,8 @@ static void setOwnerWithIdentity(id<MTLResourceSPI> resource, auto webProcessID)
 
 void Device::setUncapturedErrorCallback(Function<void(WGPUErrorType, String&&)>&& callback)
 {
+    if (m_uncapturedErrorCallback)
```

Comment:
We don't need to make a list, as there can only be one uncaptured error callback per GPUDevice according to the specification.